server/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -369,9 +369,16 @@ protected TranslogStats stats() throws IOException {
     }
 
     public void testFindEarliestLastModifiedAge() throws IOException {
-        long fixedTime = System.currentTimeMillis();
-        long[] periods = new long[10];
-        for (int i = 0; i < 9; i++) {
+        final int numberOfReaders = scaledRandomIntBetween(1, 10);
```

Comment:
Can you make this go down to 0?

Reply:
I explicitly test the 0 readers case in that test as well as a random number of readers between 1 and 10 I can do the change of you prefer however.

Reply:
oh I see. I missed it.